.travis.yml

```diff
@@ -26,5 +26,5 @@ jobs:
     - "$TRAVIS_BUILD_DIR/streamr-docker-dev/streamr-docker-dev/bin.sh start 5"
     - "$TRAVIS_BUILD_DIR/streamr-docker-dev/streamr-docker-dev/bin.sh log -f &"
     - while true; do http_code=$(curl -s -o /dev/null -w "%{http_code}" http://localhost/api/v1/users/me); if [ $http_code = 401 ]; then echo "EE up and running"; break; else echo "EE not receiving connections"; sleep 5s; fi; done
-    - while true; do http_code=$(curl -s -o /dev/null -w "%{http_code}" http://localhost/api/v1/ws); if [ $http_code = 426 ]; then echo "brokers up and running"; break; else echo "brokers not receiving connections"; sleep 5s; fi; done
+    - while true; do http_code=$(curl -s -o /dev/null -w "%{http_code}" http://localhost/api/v1/streams); if [ $http_code = 401 ]; then echo "brokers up and running"; break; else echo "brokers not receiving connections"; sleep 5s; fi; done
```

Comment:
Isn't API endpoint `http://localhost/api/v1/streams` served by E&E? You could try curling /api/v1/volume once https://github.com/streamr-dev/streamr-docker-dev/pull/39 is landed since that is a simple HTTP endpoint that should always return 200 and some JSON.

Reply:
PR has been landed